tests/nvidiagpu/deploygpu_test.go

```
@@ -106,6 +103,12 @@ var _ = Describe("GPU", Ordered, Label(tsparams.LabelSuite), func() {
 
 		BeforeAll(func() {
 			glog.V(0).Infof("Start of the test case, BeforeAll")
+
+			WorkerNodeSelector = map[string]string{
+				inittools.GeneralConfig.WorkerLabel:                          "",
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)